lib/container_tab_indicator.dart

```diff
@@ -35,63 +65,63 @@ class ContainerTabIndicator extends Decoration {
 }
 
 class _ContainerTabPainter extends BoxPainter {
-  final ContainerTabIndicator indicator;
-  EdgeInsets resolvedPadding;
-
   _ContainerTabPainter(
-    void Function() onChanged, {
-    this.indicator: const ContainerTabIndicator(),
+    VoidCallback? onChanged, {
+    this.indicator = const ContainerTabIndicator(),
   }) : super(onChanged) {
-    this.resolvedPadding = this.indicator.padding.resolve(TextDirection.rtl);
+    resolvedPadding = indicator.padding.resolve(TextDirection.rtl);
   }
 
+  final ContainerTabIndicator indicator;
+  late EdgeInsets resolvedPadding;
+
   @override
   void paint(Canvas canvas, Offset offset, ImageConfiguration configuration) {
     double width;
     double height;
 
-    if (this.indicator.widthFraction != null) {
-      width = (this.indicator.widthFraction * configuration.size.width)
-          .clamp(0, configuration.size.width);
+    assert(configuration.size != null); // todo ?!
```

Comment:
I wasn't exactly sure what to do with all those `configuration.size` which are nullable. I added an assert and used the bang operator (which I know is not ideal). What do you think?

Reply:
huh, I'll have to take a closer look at this later in the night

Reply:
Okay 👍

Reply:
I think this is fine